apollo-router/src/services/layers/static_page.rs

```diff
@@ -145,9 +116,14 @@ pub(crate) fn sandbox_page_content() -> String {
 
 #[derive(Template)]
 #[template(path = "homepage_index.html")]
-struct HomepageTemplate {}
+struct HomepageTemplate {
+    graph_ref: String,
+}
 
-pub(crate) fn home_page_content() -> String {
-    let template = HomepageTemplate {};
+pub(crate) fn home_page_content(homepage_config: Homepage) -> String {
+    let template = HomepageTemplate {
+        graph_ref: homepage_config.graph_ref.unwrap_or("".to_string()),
```

Comment:
```suggestion
        graph_ref: homepage_config.graph_ref.unwrap_or_default(),
```

Reply:
If the templating is able to handle `Option` it's even better instead of passing empty string.

Reply:
I will give this a try!

Reply:
Seems to be working 👍